test/data_scraping/test__main__.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+from unittest.mock import patch, Mock
```

Comment:
```suggestion
from unittest.mock import patch
from unittest.mock import Mock
```

Reply:
Ignoring .exe files written in -gitignore and chromedriver.exe removed from repository.

I fo not know what you mean by the banana image files however. Those are ignored by the gitignore file. How do you even know what I keep in my banana image folder locally?!?

Reply:
I have looked into the case. The change is necessary. The reason is that an image in the master repository is corrupted and cannot be opened. It is not a jpg file.

The bad file was replaced by a good file :)

Reply:
The corrupted file is there to test some functionality. I'll fix :)